feat: add redis session revoke script for specific user (#33)
betagouv/grist-utils: precise-revoke-redis-sessions/README.md

# Precisely revoke sessions in Redis

> 

Reviewer: ```suggestion

**Status: ✅ Ready to be used in production**

```

# Precisely revoke sessions in Redis

**Status: ✅ Ready to be used in production**

## Description

This script runs in `redis-cli`.
It requires access to the Redis instances used by Grist.
Because the session object in Redis is a JSON object, the script simplifies by just checking if the object contains the username.

> If the username is underspecified, it might delete sessions for other users with similar names.

Reviewer: The description can be adapted now that we match the exact username, right?

> Because the session object in Redis is a JSON object, the script simplifies by just checking if the object contains the username.
> If the username is underspecified, it might delete sessions for other users with similar names.

Author: ```suggestion
Because the session object in Redis is a JSON object, the script simplifies by just checking if the object contains the quoted username.
```

Reviewer: LGTM!
